### ISSUE TITLE
Aplicar metodologías como "design thinking" forma parte de la planificación

### ISSUE THREAD

**Issue by JJ** [moshidev/TFG-MaquiTracker#14]
Aunque en general lo que dices aquí es cierto

https://github.com/moshidev/TFG-MaquiTracker/blob/eb9c61df0c318222eed7e3a5189a22bef2478321/memoria/secciones/02_problema_a_resolver.tex#L19-L25

El área general del problema forma parte de la introducción; *design thinking* puede ayudarte no a formular, sino a definir con qué parte del problema nos vamos a quedar. Crear user journeys y a partir de ellos historias de usuaro *ya* es planificación: estás priorizando unas frente a otras solo por asignarles un orden.

**Comment by moshidev** [comment]
Cierto, estaba confundiendo conceptos. Ahora hay un capítulo dedicado a la metodología anterior al análisis del problema.
